### ISSUE TITLE
misc comments

### ISSUE THREAD

**Issue by macaodha** [udlbook/udlbook#59]
Feel free to disagree with some of the comments below. All mostly minor things.    

Ch14 Self-supervised learning is mentioned briefly in 9.3.7, it could be helpful to comment on it's relationship to unsupervised learning in this chapter.  
14.1 "captures its essential qualities (figure 1.9)" - is figure 1.10 a better reference?  
14.3 "of the 1000 possible categories". If the reader does not know about the Inception score it might not make sense where the number 1000 comes from. You do mention this in the caption of Fig 14.4, but not in the main text.   

Ch 15 "A generative adversarial network or GAN is an" -> "A generative adversarial network, or GAN, is an"  
Ch 15 "there is no easy way to calculate the probability". Could be helpful to draw the distinction between this and the probability that the discriminator outputs.   
15.1 You are using x^* to signify a synthesized training example. This notation was not used in the previous chapter.   
Fig 15.1 If the generative model is z_i + \theta and z_i is drawn from a standard normal, in expectation, should we not expect to see no synthesized data on the right, as it would be 0 + 6.7? Should \theta here be \in {0.3, 0.49, 0.67}? Maybe I'm making a simple mistake here?   
Fig 15.4 "on faces dataset" -> "on a faces dataset", "using ImageNet database" -> "using the Imagenet-1k dataset", ...  
15.2.4 broken reference on the left "Problems 15.3–??"  
15.5.1 Here you use \mathbf{c} to represent an image, but in the previous section it was an attribute vector or class label. Presumably this is intentional, but perhaps could be confusing.   
15.5.2 "An adversarial loss derives from a discriminator ...". I found this sentence hard to parse.   
15.5.2 "VGG loss". Could mention the phrase "perceptual loss" here.   

20.1 "take it as read" Have never heard that expression before.    
20.2.1 You start by saying that a model can't learn "every" function. However, then you end this section by giving an example where it was possible to memorize complete noise. Maybe clarify that you are saying that it was possible in the example shown because the finite amount of data - this is implied, but not clearly stated.   
20.2.4 RE data augmentation. If desired, you could be more concrete here e.g, reporting performance differences of models training without and without augmentation to show that augmentation is not essential component for learning on many datasets.   
Fig 20.8 Heavily compressed.   
20.4.1 batch size:learning rate -> batch size/learning rate ?  

B.3.1 Broken figure reference i.e., "figure ??"

**Comment by udlbook** [comment]
Oisin!  Thank you so much for coming through with these comments.  I REALLY appreciate it.  The last part of writing a book is a bit of a grim slog as you can imagine.  I changed almost all of them.  Should you fancy a visit to Toronto before/after CVPR, you would be very welcome to visit!
